cluster-autoscaler/expander/leastnodes/leastnodes_test.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package leastnodes
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"k8s.io/autoscaler/cluster-autoscaler/expander"
+)
+
+func TestLeastNodes(t *testing.T) {
+	e := NewFilter()
+
+	eo0 := expander.Option{Debug: "EO0", NodeCount: 2}
```

Comment:
Please use a table-driven test. This test is very simple and I know a lot of older CA code uses unittests like this, but we're (slowly) pushing towards migrating to table-driven tests as those are a standard in go.

Reply:
Can do!

Reply:
Are there any good examples I can copy from in the repo in terms of the style to follow? I'll take a look around too